packages/base-contract/src/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { DataItem } from '@0xproject/types';
+import * as _ from 'lodash';
+
+// tslint:disable-next-line:completed-docs
+export function formatABIDataItem(abi: DataItem, value: any, formatter: (type: string, value: any) => any): any {
```

Comment:
Why `value: any`? We assume it's iterable in the code body. It should be at least `any[]`?

Reply:
We only assume it's iterable in one of the branches. It might be a `BigNumber` and `BigNumber` is not a subtype of `any[]`